taxonomy_estimation_tutorial.md

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
hey @qclayssen, this is a great idea and a very useful start, but this repository is not the right place to develop this file :)

Here is my suggestion:

get a clone of our web repository (here it is: https://github.com/merenlab/web), get it working on your computer (here are the instructions: https://github.com/merenlab/web/blob/master/README.md), create a branch for anvi'o taxonomy, remove this file from this repository and start developing it in the web repository as a blog post.

You can copy-paste another blog post with a new name (such as this one: https://github.com/merenlab/web/blob/master/_posts/anvio/2016-11-08-pangenomics-v2.md), and start changing its contents. If you set up the web properly on your computer following the instructions in the README file, then you will get to render your changes in real-time to make sure things will look good on the web :)

Even if it is very drafty (or even if you don't change what is already here and only carry it there as a blog post), that is still fine. Once your branch is online I can go through it and start adding examples etc.